input/includes/resourcereferencesummary.md

```diff
@@ -0,0 +1,19 @@
+|Resource Type|Profile|Supported Target Reference|
+|---|---|---|---|
```

Comment:
Commenting here to get an inline comment. 
DocumetReference is missing from the list, please add.

Reply:
Added DocumentReference